resources/views/dashboard/training/new_ticket.blade.php

```diff
@@ -158,15 +62,15 @@
 						$currentDateET = new DateTime("now", new DateTimeZone('America/New_York') ); 
 						$currentTimeET = $currentDateET->format('H:i');
 					@endphp
-                    {!! Form::label('start', 'Start Time in Eastern (now ' . $currentTimeET . ')', ['class' => 'form-label']) !!}
+                    {!! Form::label('start', 'Start Time ET (now ' . $currentTimeET . ')', ['class' => 'form-label']) !!}
                     <div class="input-group date" id="datetimepicker2" data-target-input="nearest">
                         {!! Form::text('start', null, ['placeholder' => '00:00', 'class' => 'form-control datetimepicker-input', 'data-target' => '#datetimepicker2']) !!}
                     </div>
                 </div>
             </div>
             <div class="col-sm-3">
                 <div class="form-group">
-                    {!! Form::label('end', 'End Time in Eastern', ['class' => 'form-label']) !!}
+                    {!! Form::label('end', 'End Time ET', ['class' => 'form-label']) !!}
                     <div class="input-group date" id="datetimepicker3" data-target-input="nearest">
                         {!! Form::text('end', null, ['placeholder' => '00:00', 'class' => 'form-control datetimepicker-input', 'data-target' => '#datetimepicker3']) !!}
                     </div>
```

Comment:
I can't put a comment on it since it didn't change, but if we are going to automatically calculate the duration, let's make that field non-editable so that there isn't a discrepancy

Reply:
Yes... it's been this way for a while now. I prefer that we leave it editable - that way if something unforeseen happens a trainer can still set it how they want. The calculated duration is more of a convenience - not really a field validation.